WOFClassLib/Player.cs

```diff
@@ -57,8 +57,19 @@ public Player(string name = "Player")
         /// <returns>The number of letters matched</returns>
```

Comment:
so cool that you're using the XML comment format to document 🔥. I'm gonna try it too